.travis.yml

```diff
@@ -21,7 +21,7 @@ script: >
   -e TRAVIS=$TRAVIS -e TRAVIS_COMMIT_RANGE=$TRAVIS_COMMIT_RANGE
   -e TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST -e TRAVIS_REPO_SLUG=$TRAVIS_REPO_SLUG
   -e TRAVIS_BRANCH=$TRAVIS_BRANCH -e TRAVIS_COMMIT=$TRAVIS_COMMIT
-  -e GOPATH=/gopath -e TRASH_CACHE=/gopath/.trashcache
+  -e GOPATH=/gopath -e TRASH_CACHE=/gopath/.trashcache -e HOME=/gopath
```

Comment:
Why is this necessary? It would be nice to at least mention this in the commit message.

Reply:
dropped.  I was just trying to fix the CI but then I realized it is just the missing dependency in the Skopeo vendor.conf